.goreleaser.yml

```
@@ -15,7 +15,7 @@ builds:
   flags:
     - -trimpath
   ldflags:
-    - '-s -w -X main.version={{.Version}} -X main.commit={{.Commit}}'
+    - '-s -w -X github.com/dynatrace-oss/terraform-provider-dynatrace/provider/version.Current={{ .Version }} -X github.com/dynatrace-oss/terraform-provider-dynatrace/provider/version.OperatingSystem={{ .Os }} -X github.com/dynatrace-oss/terraform-provider-dynatrace/provider/version.CPUArchitecture={{ .Arch }}'
```

I vote for multiple entries in the array instead of one long

```suggestion
    - '-s -w'
    - '-X github.com/dynatrace-oss/terraform-provider-dynatrace/provider/version.Current={{ .Version }}'
    - '-X github.com/dynatrace-oss/terraform-provider-dynatrace/provider/version.OperatingSystem={{ .Os }}'
    - '-X github.com/dynatrace-oss/terraform-provider-dynatrace/provider/version.CPUArchitecture={{ .Arch }}'
```